src/rgw/rgw_bucket.cc

```diff
@@ -1,19 +1,22 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab ft=cpp
 
-#include <errno.h>
-
-#include <string>
+#include <cerrno>
 #include <map>
 #include <sstream>
+#include <string>
 #include <string_view>
+#include <variant>
 
 #include <boost/format.hpp>
 
+#include "common/async/blocked_completion.h"
+#include "common/async/librados_completion.h"
```

Comment:
these includes, along with `<variant>` above, appear to be unused in rgw_bucket.cc

Reply:
Fixed.